tasks/build-monthly-updates-as-json.js

```diff
@@ -149,3 +152,38 @@ function getBase64Content(file, imageRelativePath) {
 
     return base64;
 }
+
+/**
+ * Transform EE into Serenity to match PIM version.
+ *
+ * @param editions
+ */
+function transformToPimEditions(editions) {
+    return editions.map(edition => {
+        return edition === 'EE' ? 'Serenity' : edition;
+    });
+}
+
+function getNotificationEndDate(starDateString) {
+    let startDate = new Date(starDateString);
+    startDate.setDate(startDate.getDate() + 7);
```

Comment:
I think + 7 should not be hardcoded. But we should have a dynamic notification period instead, no ?

Reply:
The rule is +7 for the updates. So no need to have dynamic period. And I don't like ocnstant when it's not used anywhere else (it's simple like that)